src/Tests/Microsoft.NET.Publish.Tests/NetCorePublishItemsOutputGroupTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Publish.Tests
+{
+    public class NetCorePublishItemsOutputGroupTests : SdkTest
+    {
+        public NetCorePublishItemsOutputGroupTests(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        private static List<string> FrameworkAssemblies = new List<string>()
+        {
+            "api-ms-win-core-console-l1-1-0.dll",
+            "System.Runtime.dll",
+            "WindowsBase.dll",
+        };
+
+        [CoreMSBuildOnlyFact]

```

Comment:
Why is this for core msbuild only?
esp. if it needs to work with VS installer projects - and from within VS, there should also be tests for desktop framework msbuild..

Reply:
Ok, I changed it to just "Fact"